samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/dataaccess/api/BinaryObjectEntity.java

```diff
@@ -1,13 +1,13 @@
 package io.oasp.gastronomy.restaurant.general.dataaccess.api;
 
-import io.oasp.gastronomy.restaurant.general.common.api.BinaryObject;
-
 import java.sql.Blob;
 
 import javax.persistence.Entity;
 import javax.persistence.Lob;
 import javax.persistence.Table;
 
+import io.oasp.gastronomy.restaurant.general.common.api.BinaryObject;
```

Comment:
Why is the order of the imports changing? we should be using all the same code formatter... or has it changed on the new release of the IDE?